open-metadata-implementation/user-interfaces/ui-chassis/ui-chassis-spring/pom.xml

```diff
@@ -358,7 +358,7 @@
                             <goal>npm</goal>
                         </goals>
                         <configuration>
-                            <arguments>install polymer-cli --prefix .</arguments>
+                            <arguments>install polymer-cli --silent --prefix .</arguments>
```

Comment:
What will this silent? I would worry if we hide any version/dependency warnings? Do you have a before/after comparison?

Reply:
this is silent the install of the polymer-cli. Always latest version.

Reply:
But for dependencies (components) you are right, shouldn't be silent. I'll push the change.

Reply:
During the build, node and npm are installed locally. 
Then polymer-cli (the silent one).
Then the dependencies according to the package.json from the static folder (silent removed).
Then polymer build is called.
Then the polymer build folder is copied to the resources.

Reply:
sounds good

Reply:
will approve with update :-)

Reply:
not sure what 'with update' means :))

Reply:
I just meant when I saw your new commits - I think I replied just before you pushed the latest changes. I was just waiting for them. All looks good thanks